src/formatDate.js

```diff
@@ -50,7 +50,45 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const result = [];
+  const separateOld = fromFormat[3];
+  const separateNew = toFormat[3];
+
+  const oldData = date.split(separateOld);
+  const indDay = fromFormat.indexOf('DD');
+  const indMonth = fromFormat.indexOf('MM');
+  let indYear = fromFormat.indexOf('YY');
```

Comment:
```suggestion
  const idxDay = fromFormat.indexOf('DD');
  const idxMonth = fromFormat.indexOf('MM');
  let idxYear = fromFormat.indexOf('YY');
```
It's more common to use this short form of "index"